deploy/07_deploy_signer_bonds_escrow.ts

```diff
@@ -0,0 +1,18 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+
+  const { deploy } = deployments
```

Comment:
Fwiw I'm not sure this is any clearer than `hre.deployments.deploy(...)`. Same for the rest of the deconstruction. 

Doesn't need to change, just wanted to call attention for a second look.

Reply:
I agree it may be unnecessary, but it depends on the script's complexness. For example, here we pull out `read` and `write` functions which are not strictly related to the `deployments` but for contracts calls. I'm afraid that `deployments.read` may cause a confusion.
https://github.com/keep-network/coverage-pools/blob/eaa867eb4078eb2fb852045436d506e5eaeac30e/deploy/06_deploy_coverage_pool.ts#L7

I'll update the simple scripts where we deconstruct just the `deploy` function. And I'm waiting for your opinion on the case I mentioned above.

Reply:
I updated the code in 3a8d6796dff73bb6f77bfe8702e60c61f32b833e